### DISCUSSION TITLE
Cannot Push

### DISCUSSION THREAD

**Discussion by tms57** [community/community#61116] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Last December I went through the process of changing my primary branch from master to main.
This was all good until I tried to push some new code (a completely different page) to my personal GitHub page, which I hadn't done in 8 months.
I suspect my mistake was to simply copy the new content (including a new .git file) into my local folder the THEN attempting to push.

Now my errors are: (after running `git **push -u origin main`**)

fatal: '_user_.github.io' does not appear to be a git repository
fatal: Could not read from remote repository.

How can I fix this: delete the old content currently up on GitHub and go on with life using my new page content?

Thanks.

**Comment by nirav-gajera** [comment]
https://stackoverflow.com/questions/32238616/git-push-fatal-origin-does-not-appear-to-be-a-git-repository-fatal-could-n

https://stackoverflow.com/questions/33248886/github-does-not-appear-to-be-a-git-repository

**Reply by tms57** [reply]
I tried everything in the above links but I think somehow my GitHub url has been changed. Now I'm getting messages such as...

error: failed to push some refs to 'https://github.com/tms57/tms57.github.io.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

I'm pretty sure that tms57 wasn't repeated when I pushed successfully before : https://github.com/tms57/tms57.github.io.git
I think this is my problem, but what do I do to fix this?
Is there any way to just WIPE what is currently up on GitHub and simply PUSH the new content there?
